Cargo.toml

```diff
@@ -11,6 +11,7 @@ repository = "https://github.com/FuelLabs/fuel-vm"
 description = "FuelVM interpreter."
 
 [dependencies]
+anyhow = "1.0"
```

Comment:
This is added only for test utils and should be moved to `dev-dependencies`

Reply:
Well utils is not under only test and is publicly exported from the crate. But I should make this optional and only enable when the `test-helpers` feature is enabled.